Source/error.cpp

```diff
@@ -1,10 +1,16 @@
+/**
+ * @file error.cpp
+ *
+ * Implementation of ingame message functions.

```

Comment:
```suggestion
 * Implementation of in-game message functions.
```

Apply same change to other cases below.